Safety/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -223,3 +247,80 @@ attitudeState& FatalFailureMode::getInstance()
     return singleton;
```

Comment:
Are you not checking if arm/disarm has been set here? You would have to check in every state right, in case the switch is flicked during any state?

Reply:
This comment has been resolved. Me and Dhruv agreed that it would be acceptable to only check in fetch instructions mode because the time delay would not inhibit any safety.